### PR TITLE
Adding build of apidocs into doc build

### PR DESCRIPTION
**Pull request by adamkerz**
(won't work on readthedocs, I don't believe)

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -38,28 +34,3 @@ class intergov:
 pygments_style = 'sphinx'
 html_theme = 'alabaster'  #'default'
 html_static_path = ['_static']
-htmlhelp_basename = 'intergov_ledgerdoc'
```
Comment (monkeypants) [inline docs/conf.py:41]: good, we don't need that.

File: docs/conf.py [hunk 2]
```diff
@@ -38,28 +34,3 @@ class intergov:
...
-
-latex_elements = {
-    'papersize': 'a4paper',
-}
-latex_documents = [
-    ('index', 'intergov_ledger.tex',
-     u'Intergov Ledger Documentation',
-     u'Chris Gough', 'manual'),
-]
-
```
Comment (monkeypants) [inline docs/conf.py:51]: why kill latexpdf? is it not used by RTD?

File: docs/conf.py [hunk 3]
```diff
@@ -38,28 +34,3 @@ class intergov:
...
-texinfo_documents = [
-    ('index', 'intergov_ledger',
-     u'Intergov Ledger Documentation',
-     u'Chris Gough',
-     'intergov_ledger',
-     'One line description of project.',
-     'Miscellaneous'),
-]
```
Comment (monkeypants) [inline docs/conf.py:65]: honestly, who throws a shoe!

(nobody uses info pages or man pages for this sort of thing, good deletion)

**Review by monkeypants — APPROVED** [review]
LGTM

File: docs/conf.py (hunk 2 above)
Comment (adamkerz) [reply docs/conf.py:51]: Oh, does it. Just thought we didn't use it. I haven't finished reading (I haven't really started, not a high priority) but I think that readthedocs can't do apidoc generation for us anyway, so we would have to push autogenerated .rst to get it the rest of the way, may as well just build it as we want and tell readthedocs where to look for it. That's my current thinking...

Comment (monkeypants) [reply docs/conf.py:51]: OK, that would be fine.

I think maybe you can make slightly evil hooks in conf.py to trick it RTD into doing interesting things... but having a pre-commit generation step is probably more hygenic.

**Comment by monkeypants** [comment]
lets find out...

**Comment by monkeypants** [comment]
No, @adamkerz the RTD build is broken

first broke Friday (https://readthedocs.org/projects/trustbridge-intergov/builds/11036813/) but now suffering from
```
Configuration error:
There is a programmable error in your configuration file:

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/trustbridge-intergov/envs/latest/lib/python3.7/site-packages/sphinx/config.py", line 368, in eval_config_file
    execfile_(filename, namespace)
  File "/home/docs/checkouts/readthedocs.org/user_builds/trustbridge-intergov/envs/latest/lib/python3.7/site-packages/sphinx/util/pycompat.py", line 150, in execfile_
    exec_(code, _globals)
  File "/home/docs/checkouts/readthedocs.org/user_builds/trustbridge-intergov/checkouts/latest/docs/conf.py", line 12, in <module>
    import intergov  # noqa
ModuleNotFoundError: No module named 'intergov'
```
May have been hurt by this
```
modules = ('intergov',)
for module in modules:
   mod = os.path.join(project_root, module)
```
and this
```
- class intergov:
-    __version__ = '0.1.0'
+ import intergov  # noqa
```
in conf.py

**Comment by adamkerz** [comment]
No, well, yes. From my minimal reading, RTD doesn't do apidoc builds, we have to do it ourselves... So it's no surprise. But we need those lines to make apidoc work. Works locally fine, although autobuild doesn't refresh when code changes currently either.

**Comment by monkeypants** [comment]
I mean RTD build is broken totally. New docs are not built.

**Comment by adamkerz** [comment]
Right, gotcha...

**Pull request merged**